shell/browser/api/electron_api_screen.cc

```diff
@@ -153,9 +192,9 @@ gin::ObjectTemplateBuilder Screen::GetObjectTemplateBuilder(
       .SetMethod("getPrimaryDisplay", &Screen::GetPrimaryDisplay)
       .SetMethod("getAllDisplays", &Screen::GetAllDisplays)
       .SetMethod("getDisplayNearestPoint", &Screen::GetDisplayNearestPoint)
+      .SetMethod("screenToDipPoint", &Screen::ScreenToDIPPoint)
```

Comment:
I think this will make the API available on Mac as well. Do we want to do that, and just return the same point without any conversion, or gate away these APIs if on Mac?

Reply:
Changed to gate to Linux and Windows!